docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
       - db
     volumes:
       - .:/app
-      # - ./node_modules/eq-author-graphql-schema:/app/node_modules/eq-author-graphql-schema
+      - ./node_modules/eq-author-graphql-schema:/app/node_modules/eq-author-graphql-schema
```

Comment:
Don't forget to comment this back out. :)

Reply:
@samiwel reminder :)